### PR TITLE
[G2M] import modal and styled card container

### PR DESCRIPTION
**Pull request by parkdoeui**

### CODE REVIEW

File: src/modal-container.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+ModalContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+  maxWidth: PropTypes.string,
+}
+
+ModalContainer.defaultProps = {
+  maxWidth: 768,
+}
```
Comment (kc-leung) [inline src/modal-container.js]: Proptype doesn't match number / string? better string maybe user wants to use rem/em/px/% ??

Screen Shot 2021-04-30 at 9 17 18 AM

**Review by kc-leung — COMMENTED** [review]
I think adding a close button to the top-right is a good idea not everybody knows by clicking on the blank spaces the modal closes. Specially those who are not familiar with web application.

File: src/modal-container.js (hunk 1 above)
Comment (parkdoeui) [reply src/modal-container.js]: Good catch, it should be number rather than string

**Comment by parkdoeui** [comment]
> I think adding a close button to the top-right is a good idea not everybody knows by clicking on the blank spaces the modal closes. Specially those who are not familiar with web application.

Completely agree with this. However, I don't see the necessity to include button UI as it can be added on the implementor's end in whichever form within children. Otherwise, styling and modifying functionalities of button would be a lot difficult for end-users perspective later on. That said, modal already comes with a functionality that allows implementors to plug button's onClick events. 

Anyways, it was a nice UX feedback 👏

**Comment by woozyking** [comment]
was this ported from `react-labs`? if so i'd recommend the way that was used to preserve commit history for the first batch https://github.com/EQWorks/lumen-ui/pull/1

File: src/modal-container.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <div className={classes.root}>
+      {children}
+    </div>
+  )
+}
+
+ModalContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+  maxWidth: PropTypes.number,
```
Comment (kc-leung) [inline src/modal-container.js:30]: Will the user be able to use "em", "rem", "wh", etc if it only takes number?

**Review by woozyking — APPROVED** [review]

**Pull request merged**